### PR TITLE
pipeline: reduce tick message in-use objects

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve? 

Reduce tick message in-use objects.

Cc: https://github.com/pingcap/ticdc/issues/2212

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Related changes

 - Need to cherry-pick to the release branch

### Release note

```release-note
None
```

### CODE REVIEW

File: pkg/pipeline/message.go [hunk 1]
```diff
@@ -64,11 +64,15 @@ func BarrierMessage(barrierTs model.Ts) *Message {
 	}
 }
 
+// TickMessage is called frequently,
+// to ease GC pressure we return a global variable.
+var tickMsg *Message = &Message{
```
Comment (amyangfei) [inline pkg/pipeline/message.go:69]: Please add a comment about this var is readonly

**Review by amyangfei — APPROVED** [review]

**Review by liuzix — APPROVED** [review]

**Pull request merged**